CHANGELOG.md

```diff
@@ -52,38 +52,61 @@
 
 
 
-- **Enable Thread Instrumentation by default**
+- **Enable Thread instrumentation by default**
 
-  The configuration option `instrumentation.thread.tracing` is now enabled by default. This will allow the agent to properly monitor code that occurs inside of threads. If you are currently using custom instrumentation to start a new transaction inside of threads, this may be a breaking change, as it will no longer start a new transaction if one already exists. [PR#1767](https://github.com/newrelic/newrelic-ruby-agent/pull/1767)
+  The configuration option `instrumentation.thread.tracing` is now enabled by default. This configuration allows the agent to properly monitor code occurring inside threads. In Ruby agent 9.0, instrumented code within threads will be recorded and associated with the current transaction when the thread was created.
 
-- **Add Fiber Instrumentation**
+  This may be a breaking change if you are currently using custom thread instrumentation. New transactions inside of threads will no longer be started if one already exists. [PR#1767](https://github.com/newrelic/newrelic-ruby-agent/pull/1767)
+
+- **Add Fiber instrumentation**
 
   `Fiber` instances are now automatically instrumented similarly to `Thread` instances. This can be configured using `instrumentation.fiber`. [PR#1802](https://github.com/newrelic/newrelic-ruby-agent/pull/1802)
 
 
-- **Ruby 2.2 and 2.3 Dropped**
+- **Removed support for Ruby 2.2 and 2.3**
 
-  Support for Ruby 2.2 and 2.3 dropped with this release. They are no longer included in our test matrices and are not supported
-  for 9.0.0 and onward. [PR#1778](https://github.com/newrelic/newrelic-ruby-agent/pull/1778)
+  Ruby 2.2 and 2.3 are no longer supported by the Ruby agent. To continue using the latest Ruby Agent version, please update to Ruby 2.4.0 or above. [PR#1778](https://github.com/newrelic/newrelic-ruby-agent/pull/1778)
 
-- **Instrumentation dropped for select gems**
+- **Removed deprecated instrumentation**
 
-  Support for the following gems had been dropped. [PR#1788](https://github.com/newrelic/newrelic-ruby-agent/pull/1788)
+  Instrumentation for the following gems had been previously deprecated and has now been removed. [PR#1788](https://github.com/newrelic/newrelic-ruby-agent/pull/1788)
     - Acts As Solr
     - Authlogic
     - DataMapper
     - Rainbows
     - Sunspot
 
+  Versions of the following technologies had been previously deprecated and are no longer supported.
+
+    - CRuby: 2.2.x, 2.3.x
```

Comment:
The ruby versions removal have their own sections in the changelog, We probably don't need to list them here too.

Reply:
Right lol good call